test/symforce_function_codegen_test_data/symengine/symforce_rust_codegen_test/src/backend_test_function_float64.rs

```diff
@@ -0,0 +1,303 @@
+            *res5 = core::f64::consts::E;
+        }
+
+        if let Some(res6) = res6 {
+            *res6 = _tmp0.powf((-1_f64 / 1_f64));
+        }
+
+        if let Some(res7) = res7 {
+            *res7 = _tmp0;
+        }
+
+        if let Some(res8) = res8 {
+            *res8 = 10_f64.ln();
+        }
+
+        if let Some(res9) = res9 {
+            *res9 = core::f64::consts::PI;
+        }
+
+        if let Some(res10) = res10 {
+            *res10 = (1_f64 / 2_f64) * core::f64::consts::PI;
+        }
+
+        if let Some(res11) = res11 {
+            *res11 = (1_f64 / 4_f64) * core::f64::consts::PI;
+        }
+
+        if let Some(res12) = res12 {
+            *res12 = _tmp1;
+        }
+
+        if let Some(res13) = res13 {
+            *res13 = 2_f64 * _tmp1;
+        }
+
+        if let Some(res14) = res14 {
+            *res14 = 2_f64 / core::f64::consts::PI.powf((1_f64 / 2_f64));
+        }
+
+        if let Some(res15) = res15 {
+            *res15 = _tmp2;
+        }
+
+        if let Some(res16) = res16 {
+            *res16 = (1_f64 / 2_f64) * _tmp2;
+        }
+
+        if let Some(res17) = res17 {
+            *res17 = x.abs();
+        }
+
+        if let Some(res18) = res18 {
+            *res18 = x.sin();
+        }
+
+        if let Some(res19) = res19 {
+            *res19 = x.cos();
+        }
+
+        if let Some(res20) = res20 {
+            *res20 = x.tan();
+        }
+
+        if let Some(res21) = res21 {
+            *res21 = x.asin();
+        }
+
+        if let Some(res22) = res22 {
+            *res22 = x.acos();
+        }
+
+        if let Some(res23) = res23 {
+            *res23 = x.atan();
+        }
+
+        if let Some(res24) = res24 {
+            *res24 = x.exp();
+        }
+
+        if let Some(res25) = res25 {
+            *res25 = x.ln();
+        }
+
+        if let Some(res26) = res26 {
+            *res26 = x.sinh();
+        }
+
+        if let Some(res27) = res27 {
+            *res27 = x.cosh();
+        }
+
+        if let Some(res28) = res28 {
+            *res28 = x.tanh();
+        }
+
+        if let Some(res29) = res29 {
+            *res29 = x.floor();
+        }
+
+        if let Some(res30) = res30 {
+            *res30 = x.ceil();
+        }
+
+        if let Some(res31) = res31 {
+            *res31 = _tmp3;
+        }
+
+        if let Some(res32) = res32 {
+            *res32 = x.asinh();
+        }
+
+        if let Some(res33) = res33 {
+            *res33 = x.acosh();
+        }
+
+        if let Some(res34) = res34 {
+            *res34 = x.atanh();
+        }
+
+        if let Some(res35) = res35 {
+            *res35 = x - 5.5_f64 * (0.181818181818182_f64 * x).floor();
+        }
+
+        if let Some(res36) = res36 {
+            *res36 = x + 1.0_f64;
+        }
+
+        if let Some(res37) = res37 {
+            *res37 = 2.0_f64 * x;
+        }
+
+        if let Some(res38) = res38 {
+            *res38 = x.powf((2_f64 / 1_f64));
```

Comment:
I guess these are some of the unused parens.  It's going to be a bit annoying to remove these at the printer level; we could either add checks when printing functions for whether any of their arguments are fully wrapped in parens, and remove them (mildly annoying, mostly we may have to override a lot more things to add this behavior to inherited printers that otherwise work fine with the base classe behavior); or we could not eagerly add parens around things like rationals, and attempt to add them anywhere you're constructing a bigger expression that may have precedence issues (harder and very error-prone).

The way we'd remove them automatically is with [`cargo fix`](https://github.com/rust-lang/rustfmt/issues/3823); unfortunately it seems like there isn't a way to run that without `cargo`, which is a bit of a bummer.

I think the easiest thing is going to be to just add `#[allow(unused_parens)]` to our generated rust code

Reply:
Done